autoload/youcompleteme.vim

```diff
@@ -521,6 +541,10 @@ endfunction
 
 
 function! s:BufferTextChangedSinceLastMoveInInsertMode()
+  if pyeval( 'vimsupport.VimVersionAtLeast("7.3.867")' )
```

Comment:
since this check is repeated multiple times why don't make a variable for it like

```viml
let s:has_textchangedi = pyeval( 'vimsupport.VimVersionAtLeast("7.3.867")' )
```